.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
dear @Sajjad-Arjmand please add `.vs` directory to `.gitignore` these files should not be in git!

Reply:
ok. sorry this is my very first team project, but i'm a quick learner 😅

Reply:
It's absolutely OK! 
feel free 🙃